internal/charmstore/search.go

```diff
@@ -463,6 +463,7 @@ func (s *Store) syncSearch() error {
 			return errgo.Notef(err, "cannot index %s", rurl)
 		}
 	}
+	logger.Infof("finished sync search")
```

Comment:
thanks for that one !
We were also thinking to not stop on error (may be another PR?)